resources/istio-configuration/values.yaml

```
@@ -126,11 +126,11 @@ global:
     # these definitions aren't used by the chart, but they are required by external tool for list generation
     istio_proxyv2:
       name: "proxyv2"
-      version: "1.11.4-distroless"
+      version: "1.11.4-solo-fips-distroless"
       directory: "external/istio"
```

Line 2 will also have to be edited to
```yaml
repository: eu.gcr.io/kyma-project/external/gcr.io/istio-enterprise
```

```suggestion
      directory: "external/gcr.io/istio-enterprise"
```